packages/web-react/src/components/UNSTABLE_Truncate/useTruncateStyleProps.ts

```diff
@@ -0,0 +1,28 @@
+import { CSSProperties, ElementType } from 'react';
+import { SpiritTruncateProps } from '../../types/truncate';
+
+interface TruncateCSSProperties extends CSSProperties {
+  '--text-truncate-lines'?: number;
+}
+
+export interface TruncateStyles<T extends ElementType> {
+  classProps: string;
+  props: SpiritTruncateProps<T>;
+  styleProps: TruncateCSSProperties;
+}
+
+export function useTruncateStyleProps<T extends ElementType>(props: SpiritTruncateProps<T>): TruncateStyles<T> {
+  const { lines, ...restProps } = props;
+
+  const TruncateMultilinesClass = 'text-truncate-multiline';
```

Comment:
this class can be prefixed. Please add the prefix hook.

Reply:
EDIT - solved. Thanks